Get started docs improvements (#12230)
docker/docker.github.io: get-started/index.md

This page contains step-by-step instructions on how to get started with Docker.

If you are looking for information on how to containerize an application using your favorite language, see [Language-specific getting started guides](../language/index.md).

We also recommend the video walkthrough from DockerCon 2020.

<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/iqqDU2crIEQ?start=30" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

In this tutorial, you'll learn about creating and deploying Docker apps, including using multiple containers with a database, and using Docker Compose. You'll also deploy your containerized app to Azure.

> ## Download Docker

Reviewer: Perhaps

```suggestion
## Install Docker
```

or

```suggestion
## Download and install Docker
```

> You can download and install Docker on multiple platforms. Choose your preferred operating system below.

Reviewer: Perhaps something like;

```suggestion
This tutorial assumes you have a current version of Docker installed on your
machine. If you do not have Docker installed, you can download and install Docker.
Choose your preferred operating system below for installation instructions:
```

> - [Mac](https://desktop.docker.com/mac/stable/Docker.dmg)
> 
> - [Windows](https://desktop.docker.com/win/stable/Docker%20Desktop%20Installer.exe)
> 
> - [Linux](../engine/install/index.md)

Reviewer: Should be no blank line between bullets here.

Probably want to add a "blank" target to make sure the user doesn't navigate away from the page itself. Perhaps make the link captions more descriptive, e.g;

```suggestion
- [Download Docker Desktop for Mac](https://desktop.docker.com/mac/stable/Docker.dmg){:target="_blank" rel="noopener" class="_"}
- [Download Docker Desktop for Windows](https://desktop.docker.com/win/stable/Docker%20Desktop%20Installer.exe){:target="_blank" rel="noopener" class="_"}
- [Install docker Engine on Linux](../engine/install/index.md){:target="_blank" rel="noopener" class="_"}
```

Should these links point to the _install_ pages instead? (which contains information that may be relevant, such as system requirements);

```suggestion
- [Install Docker Desktop for Mac](../docker-for-mac/install.md){:target="_blank" rel="noopener" class="_"}
- [Install Docker Desktop for Windows](../docker-for-windows/install.md){:target="_blank" rel="noopener" class="_"}
- [Install docker Engine on Linux](../engine/install/index.md){:target="_blank" rel="noopener" class="_"}
```

Author: These pages will be linked from the onboarding page. Would add the direct download links here instead of pointing to yet another page. I can add another link to Desktop pages for for installation instructions

Reviewer: > These pages will be linked from the onboarding page. Would add the direct download links here instead of pointing to yet another page. I can add another link to Desktop pages for for installation instructions

So, we of course don't control "how" people arrive on this page (quite likely "through Google"), in which case this page is the first page they visit. From that perspective, I think it's important to provide the "install" step in context, instead of just the download links. (The installation page also, e.g., mentions that they agree to the EULA).

If we link from the on boarding page (the "designed" flow); does that flow first take them through the installation process? If so, users that would follow _that_ flow wouldn't need this section at all.
